recommender/recommender.py

```diff
@@ -41,13 +66,29 @@ class RecommenderXBlock(XBlock):
     by students; they can also vote for useful resources and flag problematic
     ones.
     """
-    default_recommendations = List(
-        help="List of default help resources", default=[], scope=Scope.content
+    seen = Boolean(
+        help="Has the interacted with the XBlock before?",
+        default=False,
+        scope=Scope.user_info
     )
-    # A list of default recommenations, it is a JSON object across all users,
+
+    recommender_version = String(
+        help="The version of this RecommenderXBlock",
```

Comment:
That seems perfectly reasonable to me. Is there a reason why you're calling it `recommender_version`, rather than just `version`? Seems like the context should make it clear what it is the version of. :)